api/orders.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const router = express.Router();
+const {
+  getAllOrders,
+  getOrderById,
+  createOrder,
+  updateOrder,
+  addProductToOrder,
+  destroyOrder,
+} = require("../db");
+const { requireUser } = require("./utils");
+
+router.get("/", async (req, res, next) => {
+  const orders = await getAllOrders();
+
+  res.send(orders);
+});
+
+router.post("/", requireUser, async (req, res) => {
```

Comment:
we don't need to requireUser - guest accounts should make orders too

Reply:
Done.